### PR TITLE
Add config to build aarch64 wheels

### PR DESCRIPTION
**Pull request by janaknat**
Travis-CI allows for the creation of aarch64 wheels.

Note:
* This needs the latest multibuild. Update the submodule to point to the latest.

### CODE REVIEW

**Comment by janaknat** [comment]
Successful build log: https://travis-ci.com/github/janaknat/pywavelets-wheels/builds/186105041

**Comment by grlee77** [comment]
Thanks @janaknat

I know multibuild [has moved away from using Rackspace](https://github.com/matthew-brett/multibuild#update-uploads-and-rackspace) for storing wheels, but haven't tried using it since that change was made. I will try to take a look later and see if just updating multibuild is sufficient for solving that or if there are any other changes needed on our end.

**Comment by janaknat** [comment]
@grlee77 One of the needs for updating multibuild was that
```
install_run $PLAT
```
in .travis.yml does not understand the PLAT=aarch64 option in older versions of multibuild. 

That command is found under travis_linux_steps.sh and was updated to recognize aarch64.
https://github.com/matthew-brett/multibuild/blob/6e9f609a33d78baaa96b92ad8d876bc90dcc7741/travis_linux_steps.sh#L126

There were other build issues with the older version which I didn't see on updating to the newer version.

Also, I ran the build again on travis-ci.org.
See build log: https://travis-ci.org/github/janaknat/pywavelets-wheels/builds/729672291

In the link I posted before, it was running in travis-ci.com.

**Comment by janaknat** [comment]
@grlee77 Were you able to check the new version of multibuild?

**Comment by grlee77** [comment]
Hi @janaknat: I just started #13 to work on switching the uploads to anaconda.org

**Comment by janaknat** [comment]
@grlee77 Thanks!

**Comment by grlee77** [comment]
Can you rebase on master now that #13 was merged?

**Comment by janaknat** [comment]
@grlee77 I've rebased to master. A large chunk of time is spent on building numpy. Is it feasible to have a dependency on numpy>=1.19.1? That is the version that started shipping aarch64 wheels.

**Comment by grlee77** [comment]
>  A large chunk of time is spent on building numpy. Is it feasible to have a dependency on numpy>=1.19.1? That is the version that started shipping aarch64 wheels.

That seems okay to me. I think that would be consistent with how the other NumPy pinnings are currently set (the pinnings are the oldest NumPy release that had x86 wheels available for a given version of Python).

**Comment by janaknat** [comment]
@grlee77 Okay, I'll update the branch to use the latest numpy.

**Comment by janaknat** [comment]
Gah! Have a typo. Resend again. Apologies.

**Review by grlee77 — APPROVED** [review]

**Comment by grlee77** [comment]
Thanks, I will go ahead and merge this and see if the new wheels end up at multibuild-wheels-staging as expected

**Comment by janaknat** [comment]
@grlee77 Thanks for merging. Is there a timeline for when the wheels will be available on PyPI?

**Comment by grlee77** [comment]
> @grlee77 Thanks for merging. Is there a timeline for when the wheels will be available on PyPI?

@rgommers: Do you have any objection to uploading the downloading the v1.1.1 AArch64 wheels from [multibuild-wheels-staging](https://anaconda.org/multibuild-wheels-staging/PyWavelets/files) and and uploading them to PyPI? I saw you were hesitant to add new wheels to an existing release for SciPy (https://github.com/scipy/scipy/issues/11170#issuecomment-681832094).

I am inclined to try it and if it causes any problems we can remove the wheels and wait until the subsequent release.

**Comment by rgommers** [comment]
Yes, I think we can try it. The risk is low, and PyWavelets is much smaller than SciPy so it's not likely this will cause problems for anyone. 

Thanks for working on this.

**Comment by grlee77** [comment]
Thanks Ralf!   @janaknat: These AArch64 wheels are now on PyPI.

**Comment by janaknat** [comment]
@grlee77 @rgommers Thanks a lot for your help!

**Pull request merged**
